PowerEditor/src/stylers.model.xml

```diff
@@ -661,7 +661,7 @@
             <WordsStyle name="FUNCTION" styleID="8" fgColor="8000FF" bgColor="FFFFFF" fontName="" fontStyle="0" fontSize="" keywordClass="type1" />
             <WordsStyle name="OPERATOR" styleID="9" fgColor="000080" bgColor="FFFFFF" fontName="" fontStyle="1" fontSize="" />
         </LexerType>
-        <LexerType name="lisp" desc="LISP" ext="">
+        <LexerType name="lisp" desc="Lisp" ext="">

```

Comment:
Lisp (historically LISP),  so please keep it as it is.
https://en.wikipedia.org/wiki/Lisp_(programming_language)